etl/steps/data/garden/who/2025-04-07/gho_smoking.meta.yml

```diff
@@ -107,4 +126,77 @@ tables:
+
+          ##### Limitations
+          In most countries, national level brand-specific price statistics were not available, thus the retail price data were collected from one supermarket/hypermarket or kiosk/newsagent/tobacconist/independent food store usually in the capital city of the country (where survey respondents were mostly located), therefore potentially not being nationally representative.
+      tobacco_ad_ban:
+        title: Bans on tobacco advertising
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          The level of bans on tobacco advertising are defined as follows: 1) no ban, or ban that does not cover national TV, radio and print media advertising; 2) ban on national TV, radio and print media advertising only; 3) ban on national TV, radio and print media, as well as on some, but not all other forms of advertising and 4) complete ban on all forms of advertising.
+        presentation:
+          grapher_config:
+            note: This indicator includes both direct advertising such as TV, radio, print media and billboard ads and indirect advertising, such as sponsorships, product placement or promotional discounts.
+      cig_afford_pct_gdp:
+        title: Affordability of cigarettes (as a share of GDP per capita)
+        unit: "%"
+        short_unit: "%"
+        description_short: |-
+          The price of 100 packs of cigarettes as a share of GDP per capita. This price is based on the most-sold brand of cigarettes in a given country and includes taxes. Lower values indicate that cigarettes are more affordable.
+      tobacco_help_quit:
+        title: Support to help quit smoking
+        unit: ""
+        short_unit: ""
+        description_short: |-
+          National support for quitting smoking or tobacco use has different levels:
+          - Countries in dark blue offer a national quitline and cover costs for nicotine replacement therapy or other services.
+          - Countries in light blue cover the costs of some services, but not all.
+          - Countries in orange offer some services, but do not cover any costs.
+      tobacco_smoke_free:
+        title: Number of smoke-free public places
+        unit: "smoke-free public places"
+        short_unit: ""
+        description_short: |-
+          Categories of indoor public places that are completely covered by smoke-free legislation. This monitors eight types of indoor public places: health care facilities, universities, other educational facilities, government buildings, indoor offices, restaurants, pubs and bars and public transport.
+        presentation:
+          grapher_config:
+            note: |-
+              A smoking ban only counts as complete if it covers all indoor areas under any circumstances and holds nationwide.
```

Comment:
I'm not sure this note clarifies anything for me. Is it referring to the last bracket? My thinking is that this list of eight public places isn't completely exhaustive, but is 8 actually equivalent to a full ban on smoking on _all_ indoor public places?